test/puppetlabs/puppetdb/generative/overrideable_generators_test.clj

```diff
@@ -0,0 +1,29 @@
+(ns puppetlabs.puppetdb.generative.overrideable-generators-test
+  (:require [clojure.test :as t]
```

Comment:
Unless there's a solid reason for this, I'd prefer to keeps this consistent with all of our other tests and just :refer :all